### DISCUSSION TITLE
Best way to open a git ignored file

### DISCUSSION THREAD

**Discussion by Joe-Zer0** [helix-editor/helix#8573] [General]
I have a folder that is in .gitignore (cdk.out).  I like that the contents of this folder don't show up in the file picker.  But occasionally I need to look at a template.yaml that get's put in the cdk.out folder.  I have two possible solutions, but they don't quite work for this.  I was curious to get others thoughts.  The first way is with `:o` this works well except I can't tab complete folder names, so I must type out cdk.out every time.  I'm not sure if this is intended or a bug.  The other thought I had is if the `<space> F` (file picker at current directory) always showed git ignored files.  When using this feature, it may be more likely the user wants to see everything in the directory and is not looking at it from a "project" standpoint.  Please let me know what you think the best approach would be.  Thanks!

**Comment by gabydd** [comment]
One option is to have a keybind to toggle the config option for example:

```
[keys.normal]
C-t = ":toggle file-picker.git-ignore"
```
Or you can just set the config option if you always want to see gitignored files in the file-picker

**Reply by Joe-Zer0** [reply]
That's an interesting idea.  I may give that a try.  What are your thoughts on the two options I mentioned above, are they both working as expected, or should I open an issue for either?
